tests/ui/suggestions/trait-impl-param-mismatched-cross-crate.stderr

```diff
@@ -0,0 +1,54 @@
+error[E0050]: method `foo` has 1 parameter but the declaration in trait `foo` has 2
+  --> $DIR/trait-impl-param-mismatched-cross-crate.rs:12:15
+   |
+LL |     fn foo(a: AliasOfI32) {}
+   |               ^^^^^^^^^^ expected 2 parameters, found 1
+   |
+   = note: `foo` from trait: `fn(i32, u32)`
+help: add the missing parameter from the trait
+   |
+LL |     fn foo(a: AliasOfI32, u32) {}
```

Comment:
Could you take the argument label from the trait as well? I.e. suggest `fn foo(a: AliasOfI32, b: u32) {}`?